package.json

```diff
@@ -23,38 +23,37 @@
     "lint:eslint": "eslint \"**/*.{ts,tsx}\" --max-warnings=0",
     "lint:tsc": "tsc",
     "lint:prettier": "prettier --cache --check .",
-    "analyze": "npx source-map-explorer 'build/static/js/*.js'"
+    "analyze": "pnpm dlx source-map-explorer \"dist/assets/*.js\" --no-border-checks"
   },
   "dependencies": {
-    "@h5web/app": "13.0.0",
-    "@h5web/h5wasm": "13.0.0",
-    "@react-hookz/web": "15.1.0",
+    "@h5web/app": "14.0.0",
+    "@h5web/h5wasm": "14.0.0",
+    "@react-hookz/web": "25.1.0",
```

Comment:
Lots of majors, but we use only `useEventListener`, which receives only a bug fix.